src/middlewared/middlewared/plugins/system_/coredump.py

```diff
@@ -21,6 +24,15 @@ async def coredumps(self):
         for line in lines:
             exe = line[exe_pos:]
             time, pid, uid, gid, sig, corefile = line[:exe_pos].rsplit(maxsplit=5)
+            cp = await run("coredumpctl", "info", pid, check=False, encoding="utf-8", errors="ignore")
```

Comment:
Move this code to the alert please, Also please cache coredumpctl result to avoid running N coredumpctl subprocesses every minute.